client/src/components/NavBar/NavBar.tsx

```diff
@@ -52,7 +52,7 @@ const NavBar = (): JSX.Element => {
       {/*Add Box parent to push other content down, can remove box to fix nav bar to top of screen*/}
       <AppBar position="sticky" className={classes.appBar}>
         <LogoHeader />
-        {loggedInUser ? userNav() : guestNav()}
+        {true ? userNav() : guestNav()}
```

Comment:
I'm not sure if I get the use of this `true` here since `true` will always be truthy. Please can you provide more explanation?

Reply:
I truly am sorry. It was one of the temporary changes I made in order to test the feature. I was going to revert the change, it just skipped my mind. Its revert now. Thank you!